base/linalg/uniformscaling.jl

```diff
@@ -24,12 +24,29 @@ one{T}(J::UniformScaling{T}) = one(UniformScaling{T})
 zero{T}(::Type{UniformScaling{T}}) = UniformScaling(zero(T))
 zero{T}(J::UniformScaling{T}) = zero(UniformScaling{T})
 
-+(J1::UniformScaling, J2::UniformScaling) = UniformScaling(J1.λ+J2.λ)
-+{T}(B::BitArray{2},J::UniformScaling{T}) = bitunpack(B) + J
-+(J::UniformScaling, B::BitArray{2})      = J + bitunpack(B)
-+(J::UniformScaling, A::AbstractMatrix)   = A + J
-+(J::UniformScaling, x::Number)           = J.λ + x
-+(x::Number, J::UniformScaling)           = x + J.λ
+(+)(J1::UniformScaling, J2::UniformScaling) = UniformScaling(J1.λ+J2.λ)
+(+){T}(B::BitArray{2},J::UniformScaling{T}) = bitunpack(B) + J
+(+)(J::UniformScaling, B::BitArray{2})      = J + bitunpack(B)
+(+)(J::UniformScaling, A::AbstractMatrix)   = A + J
+(+)(J::UniformScaling, x::Number)           = J.λ + x
+(+)(x::Number, J::UniformScaling)           = x + J.λ
+
+(+)(U::UpperTriangular, J::UniformScaling) = UpperTriangular(U.data + J)
+function (+)(U::LinAlg.UnitUpperTriangular, J::UniformScaling)
+    Unew = copy(U.data)
+    for i = 1:size(Unew, 1)
+        Unew[i,i] = 2
+    end
+    UpperTriangular(Unew)
+end
+(+)(L::LowerTriangular, J::UniformScaling) = LowerTriangular(L.data + J)
+function (+)(L::LinAlg.UnitLowerTriangular, J::UniformScaling)
+    Lnew = copy(L.data)
+    for i = 1:size(Lnew, 1)
+        Lnew[i,i] = 2
+    end
+    LowerTriangular(Lnew)
+end
```

Comment:
@carnaval They are all over the world, but extreme exclusive. I bit like

https://www.bostonglobe.com/metro/2015/06/30/tom-and-gisele-looking-join-the-country-club/ButdqQMWmHLKcQ3QGCQGaL/story.html#

Just substitute "golf" with "numerical linear algebra" and you are close.